packages/ui/components/Footer.tsx

```diff
@@ -1,5 +1,41 @@
+import { ReactNode } from 'react'
+
+import { ArrowUpRight, Code, Discord, Twitter } from '@dao-dao/icons'
+
 export interface FooterProps {}
 
+const FooterItem = ({ icon, name }: { icon: ReactNode; name: string }) => (
+  <div className="flex flex-row justify-between items-center p-2 hover:bg-background-interactive-hover rounded-md cursor-pointer">
+    <div className="flex flex-row gap-2 items-center text-text-tertiary">
+      <div>{icon}</div>
+      <div>{name}</div>
+    </div>
+
+    <ArrowUpRight className="text-text-tertiary" height={12} width={12} />
+  </div>
+)
+
 export const Footer = ({}: FooterProps) => {
-  return <p>placeholder</p>
+  return (
+    <div className="max-w-sm">
+      <div className="py-4 text-text-tertiary">
+        Website is currrently in beta. Please let the developers know if you
+        encounter any issues.
+      </div>
```

Comment:
```suggestion
      <p className="py-4 text-text-tertiary">
        Website is currrently in beta. Please let the developers know if you
        encounter any issues.
      </p>
```

Better to wrap text in `p` tags instead of `div` tags to improve accessibility.